src/sentry/models/deploy.py

```diff
@@ -0,0 +1,30 @@
+"""
+sentry.models.deploy
+~~~~~~~~~~~~~~~~~~~~
+"""
+
+from __future__ import absolute_import
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import (
+    BoundedPositiveIntegerField, FlexibleForeignKey, Model
+)
+
+
+class Deploy(Model):
+    __core__ = False
+
+    organization_id = BoundedPositiveIntegerField(db_index=True)
+    release = FlexibleForeignKey('sentry.Release')
+    environment_id = BoundedPositiveIntegerField(db_index=True)
+    date_finished = models.DateTimeField(default=timezone.now)
+    date_started = models.DateTimeField(null=True, blank=True)
+    name = models.CharField(max_length=64, null=True, blank=True)
+    url = models.URLField(null=True, blank=True)
+
+    class Meta:
+        app_label = 'sentry'
+        db_table = 'sentry_deploy'
+        unique_together = (('organization_id', 'release', 'environment_id'),)
```

Comment:
what does that mean for deploy emails though? are we just going to email everyone again saying hey this release that already was deployed to production is going out to production again? and won't the commits be the same? 

Even if we do want that behavior, my thought was that it's easier to go from strict --> less strict data constraints. if i remove this constraint and then we decide to re-add it, it'll involve another data migration to merge deploys, etc. To me, it just seemed safer to do this while we're working on the feature and see what makes sense once it's actually been built.